i18n/en.json

```diff
@@ -264,7 +264,7 @@
   "admin.billing.subscription.cancelSubscriptionSection.description": "At this time, deleting a workspace can only be done with the help of a customer support representative.",
   "admin.billing.subscription.cancelSubscriptionSection.title": "Cancel your subscription",
   "admin.billing.subscription.cloudTrial.daysLeftOnTrial": "There are {daysLeftOnTrial} days left on your free trial",
-  "admin.billing.subscription.cloudTrial.lastDay": "This is the last day of your free trial. Your access will expire at 11:59pm PT.",
+  "admin.billing.subscription.cloudTrial.lastDay": "This is the last day of your free trial. Your access will expire today at 11:59pm PT.",
```

Comment:
Now that I think about it, the 11:59 is UTC not PT, so we should probably change this. 2 suggestions:

1. We display the time in UTC
2. We parse the time into the local time and display that

Reply:
@marianunez  @annelieseklein thoughts around Nick suggestions?

Reply:
+1 to show the time in user's local time

Reply:
+1 to do that in another ticket.

Reply:
@pablovelezvidal can you create the ticket to keep track of it outside of this PR?

Reply:
@pablovelezvidal reminder on this so we don't forget

Reply:
@marianunez https://mattermost.atlassian.net/browse/MM-35191